package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "ethereum-multicall",
-  "version": "2.5.0",
+  "name": "ethereum-multicall-etl",
+  "version": "2.5.1",
```

Comment:
Please remove this package update.

Reply:
Thanks for a quick reply, all extra changes are removed